taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -1093,72 +1092,58 @@ func (self *Contract) state_put(key *common.Hash, state *State) {
 	}
 }
 
-// Creates validator and delegation based on the given values
-func (self *Contract) apply_genesis_entry(delegator_address *common.Address, transfers []GenesisTransfer) {
-	// TODO fill them?
-	var args RegisterValidatorArgs
+func (self *Contract) apply_genesis_entry(validator_info *GenesisValidator) {
```

Comment:
Thanks to your changes in config it is now possible to refactor this method as it should not have almost no custom code at all. Instead, it can reuse already existing methods. Pseudo code:

for validator : initial_validators {
  registerValidator(validator);
  
  for delegation : validator.delegations {
    delegate(validator, delegation)
  }
}

I think it should be fairly easy & quick change that we could do even in this PR ? The related issue here:
https://github.com/Taraxa-project/taraxa-evm/issues/56

Reply:
Ok. I will try this